### DISCUSSION TITLE
UPnP/DLNA Bridge VS Docker

### DISCUSSION THREAD

**Discussion by Gt1ult** [LMS-Community/slimserver#710] [General]
Dear Community, 
I have installed LMS container in docker and everything is fine.
However I have an issue to make UPnP/DLNA Bridge plug-in works. In fact UPnP/DLNA Bridge cannot detect any DLNA device. 
Somewhere I read that I have to use 'host' mode network with docker. However I do not how to set this option.
Do you have any idea how to make workable UPnP/DLNA Bridge  with docker? Thank you very much!!

**Comment by mherger** [comment]
Hi @Gt1ult - there's a much more active community on http://forums.slimdevices.com. I'd suggest you post your question there.

**Comment by Gt1ult** [comment]
> 
> 
> Hi @Gt1ult - there's a much more active community on http://forums.slimdevices.com. I'd suggest you post your question there.

Thank you for the answer!!! I solved, I had just to wait a bit to make it works, after put the LMS container in docker mode.

**Comment by jpmenil** [comment]
If using a bridge, the discovery will not work, (broadcast isn't working on docker driver bridge). Meaning that you will need to configure all your clients accordingly, or setup your docker in network mode host.
